app/helpers/react_components/track/exercise_community_solutions_list.rb

```diff
@@ -11,13 +11,18 @@ def to_s
       def request
         {
           endpoint: Exercism::Routes.api_track_exercise_community_solutions_url(exercise.track, exercise),
-          query: params.slice(*AssembleExerciseCommunitySolutionsList.keys),
+          query: search_params,
           options: { initial_data: data }
         }
       end
 
       def data
-        AssembleExerciseCommunitySolutionsList.(exercise, params)
+        AssembleExerciseCommunitySolutionsList.(exercise, search_params)
+      end
+
+      memoize
+      def search_params
+        params.permit(*AssembleExerciseCommunitySolutionsList.keys)
```

Comment:
The only challenge with this is that `permit` doesn't exist for `Hash` so this will **only** work if params is Paramaters.

Reply:
Should I be using `slice` then?